route/build/src/histVars_data.f90

```diff
@@ -125,7 +140,7 @@ SUBROUTINE aggregate(this,             & ! inout:
       integer(i4b)                       :: nHRU_input
       integer(i4b)                       :: nRch_input
       integer(i4b)                       :: ix, iRoute          ! loop indices
-      integer(i4b)                       :: idxMethod           ! temporal method index
+      integer(i4b)                       :: idxMethod             ! temporal method index
```

Comment:
Looks like this is actually messing up the white space alignment

Reply:
fixed lineup